tree/dataframe/inc/ROOT/RDF/RFilterBase.hxx

```diff
@@ -55,7 +55,7 @@ public:
                const std::vector<std::string> &prevVariations, const std::string &variation = "nominal");
    RFilterBase &operator=(const RFilterBase &) = delete;
 
-   virtual ~RFilterBase();
+   ~RFilterBase();
```

Comment:
The thing is that this constructor is already overriding the constructor of the base class, so actually `clang-tidy` suggests here:

```C++
~RFilterBase() override;
```

But then you didn't like that (see https://github.com/root-project/root/pull/11285#pullrequestreview-1090458149):

> I am not a fan of adding `override` to all destructors of inheriting classes, it seems redundant

so maybe we should use `override` for the destructors after all, because `virtual` suggests that this class is not overriding a base class, and having no keywords is also bad as you explained now